blockchain_exploration/exploration.py

```diff
@@ -89,6 +93,8 @@ def get_explorer_url_for_token_wallet(network: str, address: str, base_path: Opt
     elif network == Network.TEZOS:
         # https://tzkt.io/tz1ZMZddhgxqBMMB5KwSr6L5PDJFQf2nNwbK/tokens
         return f"{base_path}/{address}/tokens"
+    elif network == Network.SUI:
+        return f"{base_path}/not-implemented/"
```

Comment:
Would this just be something like this?
https://suiexplorer.com/address/0x1b1f598f6d3bbb8b0fbc73c5676b31322ae96f1b297f85c2607b8b66f67204fa